Makefile

```diff
@@ -41,14 +52,53 @@ shutdown_dummy:
 
 
 # Runs a clean snapshot of the Soundboard against a dummy server instance.
-.PHONY: debug
-debug:
-	$(MAVEN) clean compile exec:java --projects $(SOUNDBOARD)
-	$(MAKE) start_dummy
-	$(MAKE) shutdown_dummy
+.PHONY: snapshot
+snapshot:
+	@echo "Rebuilding $(SOUNDBOARD)..."
+	@$(MAVEN) clean compile exec:java --projects $(SOUNDBOARD)
+	@$(MAKE) run_dummy
+	@$(MAKE) stop_dummy
+
+
+.PHONY: clean_release
+clean_release: $(RELEASES)
+	@if test -d $(RELEASES);then\
+		if compgen -G "$(RELEASES)*.jar" > /dev/null; then\
+			echo "Removing jars found in $(RELEASES).";\
+			rm $(RELEASES)*.jar;\
+		fi;\
+	fi;
```

Comment:
nit: rule should really be named `clean_releases` as that is purpose.

Reply:
fixed [c1f50a1](https://github.com/riecenaidoo/soundboard-gui/pull/47/commits/c1f50a1c67aa6293f66f6cb5a275312080e464df)